# The default implementation uses a comparison against the BOOST_VERSION
# pre-compiler macro, but rather the cmake version uses an actual semver version
# string. Therefore, the VERSION_LESS comparison should be used instead.
--- a/src/cpp/CMakeLists.txt
+++ b/src/cpp/CMakeLists.txt
@@ -283,7 +283,7 @@ endif()
 # allow opt-in to using Boost.Signals2
 # TODO: remove this in RStudio v1.3 and port to signals2
 if(NOT RSTUDIO_BOOST_SIGNALS_VERSION)
-   if (Boost_VERSION LESS 106900)
+   if (Boost_VERSION VERSION_LESS 1.69.00)
       set(RSTUDIO_BOOST_SIGNALS_VERSION 1)
    else()
       set(RSTUDIO_BOOST_SIGNALS_VERSION 2)
